.ci/pytorch/test.sh

```diff
@@ -391,7 +392,21 @@ test_inductor_cpp_wrapper_abi_compatible() {
 # .github/workflows/inductor-perf-test-nightly.yml
 DYNAMO_BENCHMARK_FLAGS=()
 
-if [[ "${TEST_CONFIG}" == *dynamo_eager* ]]; then
+pr_time_benchmarks() {
+
+  pip install cirron~=0.3
```

Comment:
Let's not, please. The license is compatible so if you want to copy paste it in, that is fine by me, but a clean room implementation should also not be difficult.

Reply:
I am not quite sure I understand the comment, are complaining about the pip install or about fixing a specific version. can you clarify which one and why?
also what do you mean by  "clean room implementation"?

Reply:
I think taking on an extra third party dependency just to get instruction counts is not a good trade. Although there are some mitigating factors, e.g., we would only be using this during testing, we plan to be using the instruction counts durably over time, and I don't want to deal with "oops, cirron updated and now the instruction counts are all different and our numbers are all wrong now". It's a small enough amount of code that we should take ownership of it, properly review it, and then patch it if there are problems.

(cc @malfet)

Reply:
Clean room implementation here means implementing the dep from scratch, without referencing the original's source code, which is something you would do if it was incompatibly licensed.